package.json

```diff
@@ -17,28 +17,23 @@
   "author": "ConsenSys",
   "license": "Apache-2",
   "devDependencies": {
-    "buffer": "^3.6.0",
-    "brfs": "*",
     "chai": "*",
-    "concat-stream": "^1.5.1",
-    "create-hash": "^1.1.2",
-    "eth-lightwallet": "^1.0.1",
-    "hooked-web3-provider": "*",
-    "indexof": "*",
+    "ethereumjs-testrpc": "^3.0.0",
+    "ipfsd-ctl": "^0.14.0",
     "mocha": "^2.4.5",
-    "vinyl": "^0.5.3"
+    "truffle": "^2.0.8"
   },
   "scripts": {
     "build": "browserify lib/uportregistry.js --s uportRegistry --o dist/uportregistry.js",
-    "test": "mocha --reporter spec test/*.js"
+    "build-contract": "./node_modules/truffle/cli.js build",
+    "test": "./node_modules/mocha/bin/mocha --reporter spec test/*.js"
   },
   "dependencies": {
     "bluebird": "*",
     "browser-ipfs": "0.0.5",
     "bs58": "^3.0.0",
-    "ether-pudding": "^2.0.0",
     "ipfs-api": "2.3.2",
```

Comment:
Yeah, when you do `$ truffle build` it creates the UportRegistry.sol.js which contains everything we need.